tests/CodeGen-Binding/Xamarin.Android.JcwGen-Tests/BindingTests.cs

```diff
@@ -179,6 +179,11 @@ public void JavaAbstractMethodTest ()
 				if (e.GetType ().ToString () != "Java.Lang.AbstractMethodError")
 					throw e;
 			}
+
+			var mi = ic.GetType ().GetMethod ("MethodWithCursor");
+			var mi2 = ic.GetType ().GetMethod ("global::Test.Bindings.ICursor.MethodWithCursor");
```

Comment:
This is compiler-specific, and _will_ fail if/when the compiler changes.

For example, given:

```csharp
class T : IEnumerable {
	IEnumerator global::System.Collections.IEnumerable.GetEnumerator()
	{
		return null;
	}
}
```

When compiling the above with `mcs`, we get:

```il
.method private hidebysig newslot virtual final 
          instance class [mscorlib]System.Collections.IEnumerator 
          System.Collections.IEnumerable.GetEnumerator() cil managed
```

compare to `csc` (Roslyn):

```il
  .method private hidebysig newslot virtual final 
          instance class [mscorlib]System.Collections.IEnumerator 
          'global::System.Collections.IEnumerable.GetEnumerator'() cil managed
```

Use `mcs`, and this test will *break*. That's not a good test.